frontend/src/metabase/components/FieldValuesWidget/FieldValuesWidget.tsx

```diff
@@ -424,7 +418,7 @@ export function FieldValuesWidgetInner({
       >
         {isListMode && isLoading ? (
           <LoadingState />
-        ) : isListMode && hasListValues && multi ? (
```

Comment:
We can replace `hasListValues` with `!_.isEmpty(options)` because `isListMode` implies `shouldList(...)`

and this is the definition of `hasList`:

```
export function hasList({
  parameter,
  fields,
  disableSearch,
  options,
}: {
  parameter?: Parameter;
  fields: Field[];
  disableSearch: boolean;
  options: FieldValue[];
}) {
  return (
    shouldList({ parameter, fields, disableSearch }) && !_.isEmpty(options)
  );
}
```